.github/ISSUE_TEMPLATE/language-request.md

```diff
@@ -0,0 +1,42 @@
+---
+name: Language request
+about: I really wish Highlight.js could highlight ...
+title: PLEASE read the below carefully.
+labels: ''
+assignees: ''
+
+---
+
+First let us say that we'd also love it if Highlight.js could highlight whichever language you're about to request support for!  And there is a chance you can help make that happen! But first...
+
+...PLEASE READ THE FOLLOWING...
```

Comment:
I personally don't think that PLEASE READ banners really help, and there are two of them (another one in the title). I'd remove this line, but I understand it's subjective. Feel free to ignore.